Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ however, it has to be formatted properly to pass verification tests.
 - Static analysis warnings regarding flag enums have been suppressed in order to avoid compile-time warnings or errors.
 - Action Map and Action Tree views of the UI Toolkit based Input Action Editor now prevents deselection in both views when Escape key is pressed.
 - Input Action Asset editors Auto-save feature has been modified to trigger on focus-lost when activated instead of triggering on every modification to the asset in order to reduce impact of processing required to handle modified assets.
+- Project-wide input actions template extension changed from .inputactions to .json. 
```

Comment:
Why would a user care about this? It's an internal detail, they don't use the template.

Reply:
True but we could add something to say "so it no longer shows up in the action's selector UI as its not intended to be used directly, as its just a template for the generated project wide input actions in the project settings"

Reply:
Changed